multinet/uploaders/swagger/d3_json.yaml

```diff
@@ -0,0 +1,27 @@
+Upload a D3 JSON graph file to a collection of tables
+---
+consumes:
+  - text/plain
+
+parameters:
+  - $ref: "#/parameters/workspace"
+  - $ref: "#/parameters/table"
+  - name: data
+    in: body
+    description: Raw JSON text
+    schema:
+      type: string
```

Comment:
Can we add a richer example here?

Reply:
See https://github.com/multinet-app/multinet/pull/205#discussion_r354479906.

Reply:
50daf7efb8268f5f7a3970dcf620e9fec59df838

Reply:
Sorry, I should have been more explicit here. I meant can we add an example of the d3 data format in the data (body) parameter? The same for the nested json

Reply:
Ah ok. Yeah we can do that but the nested_json example is difficult to get together (because the examples I have are rather large, production-oriented files). Let me take a crack at this and add it in and we'll see what it looks like.

Reply:
8be5a8589f1f75f486416e91b8f484ff3c0be2f0